charts/nexus-iq/README.md

```diff
@@ -69,59 +69,60 @@ The command removes all the Kubernetes components associated with the chart and
+| `persistence.awsElasticBlockStore`             | A block for using existing AWS EBS Volumes                                                                                                                                              | `nil`                                                       |
+| `persistence.awsElasticBlockStore.volumeID`    | AWS EBS Volume to use for IQ Data                                                                                                                                                       | `nil`                                                       |
+| `persistence.awsElasticBlockStore.fsType`      | File system type for the IQ Data disk                                                                                                                                                   | `nil`                                                       |
+| `persistence.awsElasticBlockStore.logVolumeID` | AWS EBS Volume to use for IQ Logs                                                                                                                                                       | `nil`                                                       |
+| `persistence.awsElasticBlockStore.logFSType`   | File system type for the IQ Logs disk                                                                                                                                                   | `nil`                                                       |
+| `persistence.csi`                              | A YAML block for defining CSI Storage Driver configuration for the Data PV. The entire block is taken as you write it. Should support _any_ csi driver that your cluster has installed. | `nil`                                                       |
+| `persistence.logCSI`                           | A YAML block for defining CSI Storage Driver configuration for the Log PV. The entire block is taken as you write it. Should support _any_ csi driver that your cluster has installed.  | `nil`                                                       |
+| `persistence.affinity.nodeSelectorTerms`       | A YAML block for defining the affinity node selection. This block is taken as you write it.                                                                                             | `nil`                                                       |
+| `resources`                                    | Resource requests and limits for the IQ pod in the cluster.                                                                                                                             | See `values.yaml` for suggested minimum recommended values. |
+| `podSecurityContext`                           | `securityContext` for the whole pod.                                                                                                                                                    | Default matches the stock container.                        |
```

Comment:
here's the doc for the field we're fixing. the rest of the changes in this table is only whitespace.